src/styles/main.css

```diff
@@ -105,6 +105,10 @@
     @apply form-input h-12;
 }
 
+.input-counter {
+    @apply text-purple-100 uppercase font-bold float-right mt-1;
+}
+
```

Comment:
#156 #184

Is this the right way to use Tailwind?

Reply:
it is! thanks @altjohndev